arch/ARM/ARMMapping.c

```diff
@@ -401,7 +401,9 @@ static void ARM_add_reglist_reg_encoding(MCInst *MI, arm_reg reg)
 	unsigned Opcode = MCInst_getOpcode(MI);
 	// if it's r0-r13 we subtract 73 to get the respective bit num (for example r1 == 1)
 	// otherwise if it's lr or pc we add 1 for the same reason as above
-	unsigned RegBitNum = reg >= 73 ? reg - 73 : reg + 1;
+	unsigned RegBitNum = reg == ARM_REG_SP ? 13 :
+			     reg >= 73	       ? reg - 73 :
+						 reg + 1;
```

Comment:
Please use no magic numbers, replace `73` with the enum name.
Also for `ARM_REG_SP` there should be some `ARM_REG_R13 = ARM_REG_SP` at the end of the enum. So please use this one.